piff/gaussian_model.py

```diff
@@ -39,6 +39,10 @@ def fit(self, star):
         """
         import galsim
         image, weight, image_pos = star.data.getImage()
+
+        # subtract background
+        image = image - self.kwargs['background']
```

Comment:
I don't think we want this here.  This should be the job of the Input handler when it reads in the file.  In #20, I have it read in a `sky_col` and subtract that off, but we can add other ways to input the background value.  But I don't think we want to have each Model class have to deal with this detail.